karton/core/backend/direct.py

```diff
@@ -276,7 +222,7 @@ def _log_channel(logger_name: Optional[str], level: Optional[str]) -> str:
         )
 
 
-class KartonBackend(KartonBackendBase):
+class KartonBackend(KartonBackendBase, KartonBackendProtocol):
```

Comment:
You probably don't need to inherit here?

```suggestion
class KartonBackend(KartonBackendBase):
```